scripts/db_migrator.py

```diff
@@ -558,9 +575,17 @@ def version_1_0_6(self):
 
     def version_2_0_0(self):
         """
-        Current latest version. Nothing to do here.
+        Version 2_0_0.
         """
         log.log_info('Handling version_2_0_0')
+        self.migrate_port_qos_map_global()
```

Comment:
We cannot add this here because all the subsequent versions have already been taken by other branches. for eg, 202111 is at version 2_0_4. There are many migration tasks that need to be done when moving from 202012 to 202111. refer to this.
https://github.com/Azure/sonic-utilities/blob/202111/scripts/db_migrator.py

Reply:
Yes, that is also my concern. 
But, do we have to add all upcoming migration for `202012` branch to `common_migration` ? Otherwise we will run into the version issue.

Reply:
I think this is fine, because this path run for all versions below it which is what we need, correct?

Reply:
> I think this is fine, because this path run for all versions below it which is what we need, correct?

Yes, Prince, for this migration, the code is fine. I think what Neetha was concerning is how we handle the migration from `202012` image to `202111` image.
For example, we already have the migrator in branch `202111`, which version is `2_0_0`. It's for migrating `migrate_config_db_port_table_for_auto_neg()`. Now, we are going to define migrator for `202012` branch, which is also at `2_0_0`. So, when we are upgrading from `202012` to `202111`, the migrator at `v_2_0_0` will not be executed since the version in db is already `2_0_0`.
One possible solution may be https://github.com/Azure/sonic-utilities/pull/2272